x/stablestake/keeper/msg_server_add_pool.go

```diff
@@ -0,0 +1,47 @@
+package keeper
+
+import (
+	"context"
+	"errors"
+
+	errorsmod "cosmossdk.io/errors"
+	"cosmossdk.io/math"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/elys-network/elys/x/stablestake/types"
+)
+
+func (k msgServer) AddPool(goCtx context.Context, msg *types.MsgAddPool) (*types.MsgAddPoolResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+
+	params := k.ammKeeper.GetParams(ctx)
+
+	if !params.IsCreatorAllowed(msg.Sender) {
```

Comment:
Lets make it only through gov instead of list of addresses can do that

ammParams.IsCreatorAllowed was only intended for faster pool creations

Reply:
I did that initially but after a call Prashant, he suggested to use exact same way to add pools as in amm